Importer/Event/Error/InterpreterExceptionEvent.php

```diff
@@ -70,16 +70,16 @@ public function getIndex()
     /**
      * @return bool
      */
-    public function isStopped()
+    public function isAbort()
```

Comment:
`IsAborted` looks better than `IsAbort`. 
But definitely `setAbort` is better than `setAborted`.